Kitodo/src/main/java/org/kitodo/services/data/UserGroupClientAuthorityRelationService.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.services.data;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.kitodo.data.database.beans.Authority;
+import org.kitodo.data.database.beans.UserGroupClientAuthorityRelation;
+import org.kitodo.data.database.exceptions.DAOException;
+import org.kitodo.data.database.persistence.UserGroupClientAuthorityRelationDAO;
+import org.kitodo.services.ServiceManager;
+import org.kitodo.services.data.base.SearchDatabaseService;
+
+public class UserGroupClientAuthorityRelationService
+        extends SearchDatabaseService<UserGroupClientAuthorityRelation, UserGroupClientAuthorityRelationDAO> {
+
+    private final ServiceManager serviceManager = new ServiceManager();
+    private final Logger logger = LogManager.getLogger(this.getClass());
```

Comment:
Please adjust the initialisation of logger like in other classes. If this should be the new way, then it should be change over the whole project in a separate pull request.